clippy_lints/src/lib.rs

```diff
@@ -478,7 +477,6 @@ pub fn register_plugins(reg: &mut rustc_plugin::Registry) {
         if_let_redundant_pattern_matching::IF_LET_REDUNDANT_PATTERN_MATCHING,
         infinite_iter::INFINITE_ITER,
         invalid_ref::INVALID_REF,
-        is_unit_expr::UNIT_EXPR,
```

Comment:
You need to place this into the list of deprecated lints

Reply:
Done